angle_aware:
  sigma: 0.2
  delta_decrease: 10
  gamma: 0.1
  alpha: 1
  psi:
    range: [[0.3, 1.7], [0.3, 2.7]]
    density: [0.05, 0.05]
    npy_name: patrol_psi.npy
output_psi_topic: patrol_psi
output_J_topic: patrol_J